src/modules/dashboard/components/ColonyHome/ColonyMembers/MembersSubsection.tsx

```diff
@@ -0,0 +1,233 @@
+import React, { useCallback, useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import Maybe from 'graphql/tsutils/Maybe';
```

Comment:
Just thinking out loud, not a request for change or anything.

This `Maybe<T>` is `T | null | undefined` while we have another  maybe at https://github.com/joincolony/colonyDapp/blob/8196b4c44b0d24a8512dff4eb60851b8959c35b5/src/data/generated.ts#L3 which is `T | null`.

I am just wondering if it is a better idea to use that and handle the `undefined` using Typescript optional parameter for better readability.

Reply:
It's going out in my next PR anyway.